.github/workflows/ci.yml

```diff
@@ -62,15 +62,15 @@ jobs:
           path: .storybook/__image_snapshots__/__diff_output__
 
       - name: Post a comment about the visual tests (if they pass)
-        uses: unsplash/comment-on-pr@master
+        uses: unsplash/comment-on-pr@v1.3.0
```

Comment:
Setting the version explicity which is best practise but also it looks that in the latest version there was a rewrite of this action into a ruby version which is now causing failures when run on CI.

It might be in the future we look at changing this to not use a third part action for comments and follow Android in using the native GH way of doing this